packages/pg-v5/commands/pull.js

```diff
@@ -125,19 +123,58 @@ const maybeTunnel = function * (herokuDb) {
   return herokuDb
 }
 
+function spawnPipe (pgDump, pgRestore) {
+  return new Promise((resolve, reject) => {
+    pgDump.stdout.pipe(pgRestore.stdin)
+    pgDump.on('close', (code) => {
+      if (code !== 0) {
+        reject(new Error(`pg_dump errored with ${code}`))
+      }
+      pgRestore.stdin.end()
+    })
+
+    pgRestore.on('close', (code) => {
+      if (code === 0) {
+        resolve(undefined)
```

Comment:
you can just do `resolve()`

Reply:
personally I would make this a one-liner:

```
pgRestore.on('close', code => code ? reject(new Error(`...`)) : resolve())
```

but if you prefer not to do this it's fine as well